Ruby/person.rb

```diff
@@ -1,6 +1,7 @@
 require_relative './decorate'
 
 class Person < Nameable
+  @rentals = []
```

Comment:
- [ ] It would be better if we have a has-many relationship between a Person and Rentals